Cookbook.Application/ApplicationServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,19 @@
+﻿using Cookbook.Application.Services;
+using FluentValidation;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Cookbook.Application
+{
+    public static class ApplicationServiceCollectionExtensions
```

Comment:
Ezt lattad valahol hogy igy kell? `ApplicationServiceCollectionExtensions`?

Reply:
Videoban csinalta igy a srac te hogy nevezned/csinalnad?

Reply:
Pont igy, igazabol azert kerdeztem :D 2019-ben eleg uj volt meg ez az egesz dependency injection C#-ban es nem volt etul sok blog, video, javaslat rola. Elkezdtunk egy uj service-t a milestone-nal, aztan szetszedtuk tobb project-re es belefutottunk ebbe a problemaba hogy vagy minden public, implementatcio, interface, etc, vagy hogyan hook-olod fel az IServiceCollection-re. 
Akkor irtam egy one pager-t a problemarol, 3 opcioval es ez volt az egyik, meg emelett is dontottunk :D Csak meglepett hogy most ugyanezt viszont latom, de akkor ugy nez ki ez lett azota a best practice :D

Reply:
na jo bevallom igazabol olvastam a one pagered :D

Reply:
amugy azt hittem konkretan a nevre kerdezel ra h miert az :D nem arra h van kulon kis class :D